JsTetris_TCAPI/TC_tetris.js

```diff
@@ -23,10 +24,12 @@ $(document).ready(function () {
             var actor;
             if (!$(this).is(':checked')) {
                 TCActive = false;
+                tc_sendStatment_Terminate ();
```

Comment:
Heh, these are all spelled wrong, "statment" instead of "statement", I'll fix them.